packages/parrot-middleware/README.md

```diff
@@ -16,16 +19,141 @@ app.use(parrot(scenarios));
 app.listen(3000);
 ```
 
+And let's say that we are using these scenarios:
+
+```js
+// scenarios.js
+const scenarios = {
+  'has one ship': [
+    {
+      request: '/ship_log',
+      response: {
+        body: [{ name: 'The Jolly Roger', captain: 'Captain Hook' }],
+      },
+    },
+  ],
+  'has more ships': [
+    {
+      request: '/ship_log',
+      response: {
+        body: [
+          { name: 'The Jolly Roger', captain: 'Captain Hook' },
+          { name: 'The Black Pearl', captain: 'Jack Sparrow' },
+          { name: 'Flying Dutchman', captain: 'Davy Jones' },
+          { name: 'The Wanderer', captain: 'Captain Ron' },
+        ],
+      },
+    },
+  ],
+  'has a server error': [
+    {
+      request: '/ship_log',
+      response: {
+        status: 500,
+      },
+    },
+  ],
+};
+
+export default scenarios;
+```
+
 Once the server is running, the following endpoints are exposed to interact with your scenarios:
 
+### POST `/parrot/scenario`
```

Comment:
Can you add a line somewhere that explains how most likely one would want to use the chrome plugin that wraps around this API to switch scenarios?

Reply:
Added a note at the bottom of the readme :D